packages/atomic/src/components/atomic-refine-modal/atomic-refine-modal.tsx

```diff
@@ -35,15 +36,50 @@ export class AtomicRefineModal implements InitializableComponent {
   @State() public error!: Error;
 
   @Prop({reflect: true, mutable: true}) enabled!: boolean;
+  @Watch('enabled')
+  watchEnabled(enabled: boolean) {
+    const modalOpenedClass = 'atomic-modal-opened';
```

Comment:
If a CSS class needed? Or can the display be managed using JSX? e.g.

```
if (!this.enabled) {
   return;
}

return <Modal/>;
```

Reply:
The display of the component itself is managed, this is for adding a class to the body, to our users can have more control over the interface, like removing the page scrollbar `body.atomic-modal-opened`